script/build_codeowners.py

```diff
@@ -51,7 +51,7 @@
     comp = get_component(name)
     if comp is None:
         print(
-            f"Cannot find component {name}. Make sure current path is pip installed ESPHome"
+            f"Cannot find component {name}. Make sure current path is pip installed ESPHome, or pip3 uninstall esphome"
```

Comment:
Please don't change this line as a part of this PR.

The issue you faced also shows that you need to use virtualenvs for your different environments, dev or pip instlaled version

```suggestion
           f"Cannot find component {name}. Make sure current path is pip installed ESPHome"
```